apps/ledger-live-mobile/src/families/elrond/accountActions.tsx

```diff
@@ -39,11 +39,10 @@ const getMainActions = ({
    */
   const preloaded = getCurrentElrondPreloadData();
   const validators = randomizeProviders(preloaded.validators);
-
-  const screen =
-    account.elrondResources && account.elrondResources.delegations.length === 0
-      ? ScreenName.ElrondDelegationStarted
-      : ScreenName.ElrondDelegationValidator;
+  const firstTimeFlow = account.elrondResources && account.elrondResources.delegations.length === 0;
+  const screen = firstTimeFlow
```

Comment:
Generally i think it's clearer when booleans are `is` or `should` prefixed, e.g. `isFirstTimeFlow` would be super clean here (IMO)

Reply:
agree with adding `is`